src/shared/components/community/community-form.tsx

```diff
@@ -247,6 +248,27 @@ export class CommunityForm extends Component<
               </div>
             </div>
           </div>
+          {this.isAdmin && (
+            <div class="form-group row">
+              <legend class="col-form-label col-6 pt-0">
+                Is default community
```

Comment:
If you replace this with

```ts
{i18n.t("is_default_community")}
```
It'll look for the `is_default_community` key in one of the translation files. I've opened a PR adding that string [here](https://github.com/hexbear-collective/lemmy-translations/pull/1).